source/validate_capability.cpp

```diff
@@ -84,6 +84,9 @@ bool IsEnabledByExtension(ValidationState_t& _, uint32_t capability) {
       SPV_OPERAND_TYPE_CAPABILITY, capability, &operand_desc);
 
   assert(lookup_result == SPV_SUCCESS);
+  if (!lookup_result)
```

Comment:
But the previous line just asserted this would be dead.  Should the assert go away?